src/EFCore.PG/Scaffolding/Internal/NpgsqlDatabaseModelFactory.cs

```diff
@@ -217,25 +217,36 @@ public virtual DatabaseModel Create(DbConnection dbConnection, IEnumerable<strin
 
         #region Type information queries
 
+        /// <summary>
```

Comment:
I prefer to have them because even where its boilerplate, it's _documented_ to be boilerplate. 

I'm not saying we should aim to document the obvious everywhere...but in a file this large, I think it adds value to get the hover tips, even if just to verify parameter descriptions.

FWIW JetBrains has a [configuration](https://intellij-support.jetbrains.com/hc/en-us/community/posts/206834215-Is-there-any-way-to-collapse-just-comments-?page=1#community_comment_206850169) for collapsing code comments by default.